src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
```

Comment:
I think you should add field capacity

Reply:
What you mean and why do I need this? It is redudant, because I already have an DEFAULT_CAPACITY and I don`t need one more capacity field